lib/kazan/models.ex

```diff
@@ -39,7 +39,18 @@ defmodule Kazan.Models do
     cond do
       kind -> {:ok, kind}
       Map.has_key?(data, "kind") and Map.has_key?(data, "apiVersion") ->
-        version = data["apiVersion"] |> String.split("/") |> List.last
+        version = case data["apiVersion"] do
+          "v" <> v -> 
```

Comment:
TODO: This logic also needs to support the `Kazan.Models.Io.K8s.Apimachinery.Pkg.Apis.Meta.V1` namespace. Not sure what the `apiVersion` is for these objects.